pkg/lib/authn/identity/ldap/provider.go

```diff
@@ -100,3 +109,35 @@ func sortIdentities(is []*identity.LDAP) {
 		return is[i].CreatedAt.Before(is[j].CreatedAt)
 	})
 }
+
+func (p *Provider) CreateNormalizedIdentitySpecFromLDAPEntry(serverConfig *config.LDAPServerConfig, entry *ldap.Entry) (*identity.Spec, error) {
```

Comment:
Rename this to `MakeSpecFromEntry(serverConfig *config.LDAPServerConfig, entry *ldap.Entry) (*identity.Spec, error)`